spec/models/piece_spec.rb

```diff
@@ -19,6 +19,30 @@
       # it is obstructed because there is a piece in position row: 1, col: 2
     end
 
+    it 'returns false for pieces with no vertical inbetween' do
+      game = FactoryGirl.create(:game) # Would do this using FactoryGirl
+      piece = game.pieces.find_by_current_row_index_and_current_column_index(0, 0)
+
+      expect(piece.obstructed?(1, 0)).to eq false
+      # it is obstructed because there is a piece in position row: 1, col: 2
+    end
+
+    it 'returns false for pieces with no diagnal inbetween' do
+      game = FactoryGirl.create(:game) # Would do this using FactoryGirl
+      piece = game.pieces.find_by_current_row_index_and_current_column_index(0, 0)
+
+      expect(piece.obstructed?(1, 1)).to eq false
+      # it is obstructed because there is a piece in position row: 1, col: 2
+    end
+
+    it 'returns false for pieces with no horizontal inbetween' do
+      game = FactoryGirl.create(:game) # Would do this using FactoryGirl
+      piece = game.pieces.find_by_current_row_index_and_current_column_index(0, 0)
+
+      expect(piece.obstructed?(0, 1)).to eq false
+      # it is obstructed because there is a piece in position row: 1, col: 2
+    end
```

Comment:
For these three tests. I think the comments need to be updated?

when obstructed? == false. I think that means "the move is not obstructed". But the comment says "it is obstructed"